tests/framework/e2e/cluster.go

```diff
@@ -524,10 +523,3 @@ func (epc *EtcdProcessCluster) Close() error {
 	epc.lg.Info("closed test cluster.")
 	return err
 }
-
-func (epc *EtcdProcessCluster) WithStopSignal(sig os.Signal) (ret os.Signal) {
```

Comment:
If you remove this method, then it doesn't implement interface [EtcdProcess](https://github.com/etcd-io/etcd/blob/b886bbc89f31d7ac59bdfbb3d14a4263dae85a1b/tests/framework/e2e/etcd_process.go#L48) anymore.

Reply:
It seems OK. You do not remove `(*EtcdServerProcess) WithStopSignal`

Reply:
```
 stderr: # go.etcd.io/etcd/tests/v3/framework/e2e
Error: stderr: framework/e2e/cluster_proxy.go:105:19: p.etcdProc.WithStopSignal undefined (type EtcdProcess has no field or method WithStopSignal)
Error: stderr: framework/e2e/cluster_proxy.go:158:16: pp.proc.StopSignal undefined (type *expect.ExpectProcess has no field or method StopSignal)
Error: stderr: framework/e2e/cluster_proxy.go:159:9: pp.proc.StopSignal undefined (type *expect.ExpectProcess has no field or method StopSignal)
```